flycheck.el

```diff
@@ -6182,6 +6183,43 @@ Requires DMD 2.066 or newer.  See URL `http://dlang.org/'."
             (or "Warning" "Deprecation") ": " (message) line-end))
   :modes d-mode)
 
+(defun flycheck-parse-elixir-dogma(output checker buffer)
+  "Parse dogma errors from JSON OUTPUT.
+
+CHECKER and BUFFER denoted the CHECKER that returned OUTPUT and
+the BUFFER that was checked respectively.
+
+See URL `https://github.com/lpil/dogma' for more information
+about dogma."
+  (cl-flet ((alist-get (key alis) (cdr (assoc key alis))))
```

Comment:
Could you probably write this function without this local alist-get function?  It's not a bad idea per se, not at all, but it's a style that we dont use in Flycheck, so for the sake of consistency I'd prefer if we could go without local functions here ☺️